barter-data/src/provider/databento/mod.rs

```diff
@@ -0,0 +1,33 @@
+use barter_instrument::Side;
+use databento::dbn::Side as DbSide;
+use serde::{Deserialize, Serialize};
+
+#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Deserialize, Serialize)]
+pub enum DatabentoSide {
+    #[serde(alias = "buy", alias = "BUY", alias = "b")]
+    Buy,
+    #[serde(alias = "sell", alias = "SELL", alias = "s")]
+    Sell,
+}
+
+
+impl From<DbSide> for DatabentoSide {
+    fn from(value: DbSide) -> Self {
+        match value {
+            DbSide::Bid => DatabentoSide::Buy,
+            DbSide::Ask => DatabentoSide::Sell,
+            _ => {
```

Comment:
What other variants are there?

Reply:
DbSide::None => Has no purpose, but i think they use it for their snapshot feed